COCOAPODS.md

```diff
@@ -1,150 +1,381 @@
+    ```
+    
+    </div>
+
+2. Install the [`cocoapods-generate`](https://github.com/square/cocoapods-generate) plugin.
+
+    <div class="sample" markdown="1" theme="idea" mode="ruby" data-highlight-only>
+    
+    ```ruby
+    $ sudo gem install cocoapods-generate
+    ```
+    
+    </div>
+    
+3. In `build.gradle.kts` (or `build.gradle`) of your IDEA project, apply the CocoaPods plugin as well as the Kotlin
+ Multiplatform plugin it is based on.
 
-> __Note:__ The plugin is based on the multiplatform project model and requires applying the
-`org.jetbrains.kotlin.multiplatform` plugin. See details about the multiplatform plugin at
-the [corresponding page](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html).
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    plugins {
+       kotlin("multiplatform") version "{{ site.data.releases.latest.version }}"
+       kotlin("native.cocoapods") version "{{ site.data.releases.latest.version }}"
+    }
+    ```
+    
+    </div> 
+
+4. Configure `summary`, `homepage`, and `frameworkName`of the `Podspec` file in the `cocoapods` block.  
+`version` is a version of the Gradle project.
+
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    plugins {
+        kotlin("multiplatform") version "{{ site.data.releases.latest.version }}"
+        kotlin("native.cocoapods") version "{{ site.data.releases.latest.version }}"
+    }
+    
+    // CocoaPods requires the podspec to have a version.
+    version = "1.0"
+    
+    kotlin {
+        cocoapods {
+            // Configure fields required by CocoaPods.
+            summary = "Some description for a Kotlin/Native module"
+            homepage = "Link to a Kotlin/Native module homepage"
+    
+            // You can change the name of the produced framework.
+            // By default, it is the name of the Gradle project.
+            frameworkName = "my_framework"
+        }
+    }
+    ```
+    
+    </div>
+    
+5. Re-import the project.
 
 When applied, the CocoaPods plugin does the following:
 
-1. Adds both `debug` and `release` frameworks as output binaries for all iOS and macOS targets.
-2. Creates a `podspec` task which generates a [podspec](https://guides.cocoapods.org/syntax/podspec.html)
-file for the given project.
+* Adds both `debug` and `release` frameworks as output binaries for all macOS, iOS, tvOS, and watchOS targets.
+* Creates a `podspec` task which generates a [Podspec](https://guides.cocoapods.org/syntax/podspec.html)
+file for the project.
 
-The podspec generated includes a path to an output framework and script phases which automate building
-this framework during a build process of an Xcode project. Some fields of the podspec file can be
-configured using the `kotlin.cocoapods { ... }` code block.
+The `Podspec` file includes a path to an output framework and script phases which automate building
```

Comment:
The `Podspec` file includes a path to an output framework and script phases that automate building this framework during the build process of an Xcode project.

Reply:
✅